server/mdm/assets/assets.go

```diff
@@ -82,6 +82,15 @@ func ABMToken(ctx context.Context, ds fleet.MDMAssetRetriever) (*nanodep_client.
 		return nil, fmt.Errorf("loading ABM assets from the database: %w", err)
 	}
 
+	// TODO(mna): this is what it should now use, but if I change it immediately,
+	// everything else breaks. This should be changed only after the ABM API work
+	// (and then fleet.MDMAssetABMTokenDeprecated will have to be removed from
+	// the call to GetAllMDMConfigAssetsByName above).
+	//abmTok, err := ds.GetABMTokenByOrgName(ctx, abmOrgName)
+	//if err != nil {
+	//	return nil, fmt.Errorf("get ABM token by name: %w", err)
+	//}
```

Comment:
This is the part that will be punted to the cleanup ticket (or at least post-ABM API changes).